pkgs/development/go-modules/generic/default.nix

```
@@ -55,11 +58,23 @@
 with builtins;
 
 assert goPackagePath != "" -> throw "`goPackagePath` is not needed with `buildGoModule`";
+assert (vendorSha256 == "_unset" && vendorHash == "_unset") -> throw "either `vendorHash` or `vendorHash` is required";
```

throw has `vendorHash` twice.

```suggestion
assert (vendorSha256 == "_unset" && vendorHash == "_unset") -> throw "either `vendorHash` or `vendorSha256` is required";
```

---

good that you're reviewing this. apparently I've only been working on this half-asleep :grimacing: